diff --git a/node_modules/react-native-nfc-passport-reader/android/src/main/java/com/nfcpassportreader/NfcPassportReader.kt b/node_modules/react-native-nfc-passport-reader/android/src/main/java/com/nfcpassportreader/NfcPassportReader.kt
index 0000000..1111111 100644
--- a/node_modules/react-native-nfc-passport-reader/android/src/main/java/com/nfcpassportreader/NfcPassportReader.kt
+++ b/node_modules/react-native-nfc-passport-reader/android/src/main/java/com/nfcpassportreader/NfcPassportReader.kt
@@ -19,7 +19,7 @@ import org.jmrtd.lds.icao.DG2File
 import org.jmrtd.lds.iso19794.FaceImageInfo
 
 class NfcPassportReader(context: Context) {
   private val bitmapUtil = BitmapUtil(context)
   private val dateUtil = DateUtil()
 
   fun readPassport(isoDep: IsoDep, bacKey: BACKeySpec, includeImages: Boolean): NfcResult {
-    isoDep.timeout = 10000
+    isoDep.timeout = 800
     val cardService = CardService.getInstance(isoDep)
     cardService.open()
