diff --git a/node_modules/react-native-nfc-epassport-reader/ios/epassport/AppDelegate.m b/node_modules/react-native-nfc-epassport-reader/ios/epassport/AppDelegate.m
--- a/node_modules/react-native-nfc-epassport-reader/ios/epassport/AppDelegate.m
+++ b/node_modules/react-native-nfc-epassport-reader/ios/epassport/AppDelegate.m
@@ -53,7 +53,7 @@ static void InitializeFlipper(UIApplication *application) {
 - (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
 {
 #if DEBUG
-  return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index" fallbackResource:nil];
+  return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index"];
 #else
   return [[NSBundle mainBundle] URLForResource:@"main" withExtension:@"jsbundle"];
 #endif
